backend/app.ts

```diff
@@ -43,7 +44,7 @@ const processError = (response, error): void => {
 }
 
 app.get('/users', (req, res) => {
```

Comment:
maybe rename the endpoint to /players? 🤔

Reply:
I will handle renaming in future PR - I left it like this for now to avoid changes in FE.